### PR TITLE
V2.6.0

### PR DESCRIPTION
**Pull request by Ricardo-Osorio**
Update the openapi specs which now include the field hostname for both volume primary deployment and its replicas and also the field attachedOnHost for the volume.
Force the open api generator version used to 4.3.1.

### CODE REVIEW

**Comment by Ricardo-Osorio** [comment]
Bumped the version of the openapi specs to 2.6.0 and the version of the generating tool used to match 4.3.1 which is included in the generated files as a comment & property

**Review by Mojachieee — APPROVED** [review]
LGTM - but I also co-authored the change, so we should wait for another review

**Review by mhmxs — APPROVED** [review]
💯

**Pull request merged**
